_rules/role-attribute-valid-value-674b10.md

```diff
@@ -80,23 +69,34 @@ Further reading:
 This `role` attribute contains one token, and this token is a valid [WAI-ARIA role][].
 
 ```html
-<input type="text" role="textbox" />
+<label>Search: <input type="text" role="searchbox" placeholder="Enter 3 or more characters"/></label>
 ```
 
 #### Passed Example 2
 
 This `role` attribute contains two tokens, and these tokens are both valid [WAI-ARIA roles][wai-aria role].
 
 ```html
-<span role="button link" onclick="location.href='https://act-rules.github.io/'">ACT rules</span>
+<style>
+	.button {
+		border: 0.1em solid black;
+	}
+	.button:hover {
+		background-color: black;
+		color: white;
+	}
+</style>
+<span class="button" tabindex="0" role="button link" onclick="location.href='https://act-rules.github.io/'"
```

Comment:
I don't much like this example. I don't think button is a particularly appropriate role for this. I think we should show an appropriate use of fallback roles if we want them demonstrated. Maybe use on of the DPUB roles, with `link` as a fallback?

Reply:
I don't know what this example was supposed to show (I've only added the styling), but I definitely like the DPUB idea 😀